### PR TITLE
Aladdin generated examples for Automation

### PR DESCRIPTION
**Pull request by aladdindoc**
- Add Aladdin crafted examples.

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by aladdindoc** [comment]
@mikefrobbins @dcaro Can you review it?

File: src/Automation/Automation/help/New-AzAutomationAccount.md [hunk 1]
```diff
@@ -27,12 +27,21 @@ of other Automation accounts. Automation resources include runbooks, Desired Sta
...
 This command creates a new automation account named ContosoAutomationAccount in the East US region.
 
+### Example 2
+
+Creates an Automation account. (autogenerated)
+
+<!-- Aladdin Generated Example -->
+```powershell
+New-AzAutomationAccount -Location 'East US' -Name 'ContosoAutomationAccount' -ResourceGroupName 'ResourceGroup01' -Tags <IDictionary>
+```
```
Comment (dcaro) [inline src/Automation/Automation/help/New-AzAutomationAccount.md:43]: This example does not provide much value

**Pull request merged**